### ISSUE TITLE
PS Preview 6/7 generate: Could not load type 'System.Management.Automation.Subsystem.PredictionResult'

### ISSUE THREAD

**Issue by sc0ttwad3** [PowerShell/PSReadLine#2627]
## Environment
```
### Environment
PSReadLine: 2.2.0-beta2
PowerShell: 7.2.0-preview.7
OS: Microsoft Windows 10.0.22000
BufferWidth: 102
BufferHeight: 49

Last 0 Keys
```

## Exception
```
### Exception
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset() in Microsoft.PowerShell.PSReadLine2.dll:token 0x60004ca+0x0
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset() in Microsoft.PowerShell.PSReadLine2.dll:token 0x600049b+0x7
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics) in Microsoft.PowerShell.PSReadLine2.dll:token 0x6000101+0x118
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken) in Microsoft.PowerShell.PSReadLine2.dll:token 0x60000fa+0x60
```

## Steps to reproduce

Open a new PowerShell 7 - Preview 6 or preview 7 - terminal. (Preview 5 and older operate normally, with no exception).

## Expected behavior

Expected command-prompt.

## Actual behavior

No command-prompt, just the Environment and Exception information above is displayed.

**Comment by sc0ttwad3** [comment]
Using PowerShell 7.1.3 up to 7.2 preview 5, all using the same version or PSReadLine: 2.2.0-beta2 do not throw an exception.

```
PS version: 7.1.3
PSReadline version: 2.2.0-beta2
os: 10.0.22000.1 (WinBuild.160101.0800)
PS file version: 7.1.3.0
HostName: ConsoleHost (Windows Terminal)
BufferWidth: 102
BufferHeight: 49

PS C:\Users\scott>
```

**Comment by SteveL-MSFT** [comment]
This is fixed in 2.2-beta3
